packages/sorting/src/helpers/sortable-from-resolver.helper.spec.ts

```diff
@@ -42,7 +42,7 @@ describe("SortableFromResolver", () => {
     ).toThrow(SortingFieldDoesntExistError)
   })
 
-  it("Should return array of SortingField", () => {
+  it("Should return array of SortingField (without nulls)", () => {
```

Comment:
I'd put that in a (Without null values) `describe`

Reply:
done